README.md

```diff
@@ -145,74 +152,27 @@ sumAgg := view.SumAggregation{}
 meanAgg := view.MeanAggregation{}
 ```
 
-### Creating, registering and unregistering a view
-
-Create and register a view:
+Here we create a view with the DistributionAggregation over our Measure.
+Setting tags to nil (as in this example) means that Measurements with all tags
+will be aggregated together (no grouping by tag):
 
 [embedmd]:# (stats.go view)
 ```go
-v, err := view.New(
-	"my.org/video_size_distribution",
-	"distribution of processed video size over time",
-	nil,
-	videoSize,
-	distAgg,
-)
-if err != nil {
-	log.Fatalf("cannot create view: %v", err)
-}
-if err := view.Register(v); err != nil {
-	log.Fatal(err)
+v := &view.View{
+	Name:        "my.org/video_size_distribution",
+	Description: "distribution of processed video size over time",
+	MeasureName: videoSize.Name(),
+	Aggregation: view.DistributionAggregation([]float64{0, 1 << 32, 2 << 32, 3 << 32}),
 }
 ```
 
-Find view by name:
-
-[embedmd]:# (stats.go findView)
-```go
-v = view.Find("my.org/video_size_distribution")
-if v == nil {
-	log.Fatalln("view not found")
-}
-```
-
-Unregister view:
-
-[embedmd]:# (stats.go unregisterView)
-```go
-if err = view.Unregister(v); err != nil {
-	log.Fatal(err)
-}
-```
-
-Configure the default interval between reports of collected data.
-This is a system wide interval and impacts all views. The default
-interval duration is 10 seconds. Trying to set an interval with
-a duration less than a certain minimum (maybe 1s) should have no effect.
-
-[embedmd]:# (stats.go reportingPeriod)
-```go
-view.SetReportingPeriod(5 * time.Second)
-```
-
-### Recording measurements
-
-Recording usage can only be performed against already registered measure
-and their registered views. Measurements are implicitly tagged with the
-tags in the context:
-
-[embedmd]:# (stats.go record)
-```go
-stats.Record(ctx, videoSize.M(102478))
-```
-
 ### Retrieving collected data for a view
 
 Users need to subscribe to a view in order to retrieve collected data.
 
 [embedmd]:# (stats.go subscribe)
 ```go
-if err := v.Subscribe(); err != nil {
+if err := view.Subscribe(v); err != nil {
```

Comment:
This feels very odd to me now.

view.Subscribe(a view).

I don't know if view.Subscribe(...) adds much.

Reply:
I don't know what you mean here. Do you want views to be "automatically" subscribed? Or do you want to keep the method as we previously had.

This situation (a single view argument) is not the strong suite of this current proposed approach. It works much better when there are multiple views (see ochttp/example_test.go)

Reply:
Take a look at 8674a4473ec6bfe28077dc45ab4843daa053feb4